tests/m2m_through/tests.py

```diff
@@ -24,7 +24,7 @@ def test_retrieve_intermediate_items(self):
         Membership.objects.create(person=self.jim, group=self.rock)
         Membership.objects.create(person=self.jane, group=self.rock)
 
-        expected = ['Jane', 'Jim']
+        expected = ['Jim', 'Jane']
```

Comment:
`Membership` has `ordering = ('date_joined', ...)` and `jim.date_joined` is first.